src/components/lists/process-list.njk

```diff
@@ -0,0 +1,116 @@
+<section class="grid-container margin-top-4">
+  <ol class="usa-process-list">
+    <li class="usa-process-list__item">
+      <h4 class="usa-process-list__heading">Process List Item</h4>
+      <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien.</p>
+      <ul>
+        <li>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis.</li>
+        <li>Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum.</li>
+        <li>Aliquam erat volutpat. Sed quis velit.</li>
+      </ul>
+    </li>
+    <li class="usa-process-list__item">
+      <h4 class="usa-process-list__heading">Process List Item</h4>
+      <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien.</p>
+      <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien.</p>
+    </li>
+    <li class="usa-process-list__item">
+      <h4 class="usa-process-list__heading">Process List Item</h4>
+      <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien.</p>
+    </li>
+    <li class="usa-process-list__item">
+      <h4 class="usa-process-list__heading">Process List Item</h4>
+      <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Morbi commodo, ipsum sed pharetra gravida, orci magna rhoncus neque, id pulvinar odio lorem non turpis. Nullam sit amet enim. Suspendisse id velit vitae ligula volutpat condimentum. Aliquam erat volutpat. Sed quis velit. Nulla facilisi. Nulla libero. Vivamus pharetra posuere sapien.</p>
+    </li>
+    <li class="usa-process-list__item">
```

Comment:
For each of the examples, maybe use just three list items?  When this goes into the documentation site, it's might end up being a pretty long example.

Reply:
Good call, I have adjusted this.

Reply:
I cut them down a bit more